Source/u_mapinfo.h

```diff
@@ -66,7 +66,8 @@ extern umapinfo_t default_mapinfo;
 extern boolean EpiCustom;
 mapentry_t *G_LookupMapinfo(int episode, int map);
 
-int U_ParseMapInfo(boolean is_default, const char *buffer, size_t length);
-void U_FreeMapInfo();
+#define check_field(s) (s && s[0] && strcmp(s, "-"))
```

Comment:
Should we somehow "namespace" this macro, e.g. `u_check_field()`?

Reply:
Maybe we should turn it into a function, like this:
```c
boolean U_CheckField(const char *str)
{
  return str && str[0] && strcmp(str, "-");
}
```

Reply:
Right, that's nicer!